src/website/src/pages/ec/components/hero-banner/docs/usage.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A Hero banner displays a prominent message and related optional actions.
```

Comment:
```suggestion
  A Hero banner displays a prominent message and related optional actions.
```
A Hero banner is an effective approach to communicate with users, an eye-catching visual experience on top of the webpage. It delivers the most important message of the website to users.

Reply:
It can't be described as a  "large eye catching image" given the image is optional.
"when the critical site functions are preserved", I don't understand.